### PR TITLE
my attempt at doing task 6.

### PR DESCRIPTION
**Pull request by AlexGopal**
I was unable to complete it. I used my simplest code as the code to commit.
I had other solutions but I feel they relied too heavily on chatgpt and i didn't really understand how they worked.

### CODE REVIEW

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-8)

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

File: features/layout/sidebar-navigation/sidebar-navigation.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export function SidebarNavigation() {
           {/* eslint-disable-next-line @next/next/no-img-element */}
           <img
             src={
-              isSidebarCollapsed
+              isSidebarCollapsed && window.innerWidth > 1024
```
Comment (jkettmann) [inline features/layout/sidebar-navigation/sidebar-navigation.tsx:44]: This is a simple solution that works fine except when the user changes the window size or rotates the mobile device. The problem is that the size change of the browser window doesn't trigger a re-render of the React app and thus this code wouldn't execute. Taking that into account alternative approaches as in your other PR work better.

**Pull request merged**